### DISCUSSION TITLE
Who uses LinguiJS?

### DISCUSSION THREAD

**Discussion by tricoder42** [lingui/js-lingui#1404] [General]
**Do you use LinguiJS in production or for hobby project? Please let us know so we can add you to the [showroom](https://lingui.js.org//misc/showroom.html)!**

- If you want, feel free to edit the [documentation page](https://github.com/lingui/js-lingui/blob/master/docs/misc/showroom.rst) directly and send a PR.
- Otherwise please provide:
  1. Your company name
  2. Link to your website
  3. Does the website is open sourced? If so, please share a link.
  4. What's the size of the website? How many languages do you use, how many message do you have?

**Comment by MirzayevFarid** [comment]
- We’re using LinguiJS at [Radity](https://radity.com/)

- ![logo-midnight@3x](https://user-images.githubusercontent.com/29518613/91811146-173a9a00-ec40-11ea-9d20-981cfcf59877.png)

- https://radity.com
- Radity develops digital products and build web and mobile applications (20 Team members)

Thanks for such an amazing work!

**Comment by tricoder42** [comment]
Hey @MirzayevFarid! Good to hear you find this project useful 👍 

Feel free to add your project to the documentation and send a PR (like #750)

**Comment by tolunayakbulut** [comment]
- Global CTO Forum
- https://globalctoforum.org/
- No, it is not open source.
- We have 10 developer in our company.

Global CTO Forum connects world's leading CTOs and technology stakeholders.

![global-cto-forum](https://user-images.githubusercontent.com/31761883/101293116-ad282700-3824-11eb-871c-8aa0648b9701.png)

**Comment by tricoder42** [comment]
Hey @tolunayakbulut! Glad to hear you use and like LinguiJS! Feel free to send PR directly like #806
